src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,30 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import classNames from 'classnames';
 import { Loader } from '../Loader';
+interface Props {
+  todo: Todo;
+  onIconClick: () => void;
+}
+export const TodoModal: React.FC<Props> = ({ todo, onIconClick }) => {
+  const [user, setUser] = React.useState<User | undefined>();
+  const [isLoading, setIsLoading] = React.useState(false);
+  const getUserData = () => {
+    setIsLoading(true);
+    getUser(todo.userId)
```

Comment:
Ensure that the `getUser` function is correctly implemented in the `./api` file and returns the expected data format for the user.